src/ui/table.js

```diff
@@ -10,28 +10,48 @@
 //   2, 5, 6, 7, 10, 11, 13, 15, 16, 19, 21, 25, 27, 28, 29, 31, 32, 37, 40, 42,
 //   43, 45, 46, 48,
 // ];
-const tableArr2 = new Array(49);
-const redArr2 = [];
-tableArr2.fill(-1);
-function arrayFill(startElem) {
-  for (let i = startElem; i <= 24 + startElem; i++) {
-    let n = Math.floor(Math.random() * 49);
-    while (tableArr2[n] >= 0) {
-      n = Math.floor(Math.random() * 49);
-    }
-    tableArr2[n] = i;
+
+//////////////////var1/////////////
+// const tableArr2 = new Array(49);
+// const redArr2 = [];
+// tableArr2.fill(-1);
+// function arrayFill(startElem) {
+//   for (let i = startElem; i <= 24 + startElem; i++) {
+//     let n = Math.floor(Math.random() * 49);
+//     while (tableArr2[n] >= 0) {
+//       n = Math.floor(Math.random() * 49);
+//     }
+//     tableArr2[n] = i;
+//   }
+//   for (let i = startElem; i <= 23 + startElem; i++) {
+//     let n = Math.floor(Math.random() * 49);
+//     while (tableArr2[n] >= 0) {
+//       n = Math.floor(Math.random() * 49);
+//     }
+//     tableArr2[n] = i;
+//     redArr2.push(n);
+//   }
+// }
+// //to start test with 1 - arrayFill(1), to start test with 2 - arrayFill(2) etc.
+// arrayFill(2);
+///////var2////////////
+let blackRedArr = [];
+// let startElem = userObject.tableNumber;
+// console.log(startElem, typeof startElem);
+function arrayFill(startEl) {
```

Comment:
I would call it `getBlackRedArray` and move into utils folder, same for shuffle, let's call it `getShuffledArray`. You can also add description to those functions if you wish https://stackoverflow.com/questions/73219300/how-can-i-add-a-description-to-a-function-in-javascript